cypress/support/giftCard.po.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="Cypress" />
+import { BaseComponent } from './base.component';
+
+export class GiftCardPo extends BaseComponent {
+    pageUrl = 'dp';
+
+    button50Dollars = '#a-autoid-21';
```

Comment:
autoid - looks like this id generated automatically, so need to find some better one, like:
cy.get('#gc-amount-mini-picker .a-list-item').contains(expectedPrice).as('price');